sdk/nodejs/appservice/zMixins_http.ts

```diff
@@ -122,19 +122,21 @@ export class HttpEventSubscription extends mod.EventSubscription<mod.Context<Htt
         const { resourceGroupName, location } = mod.getResourceGroupNameAndLocation(args, undefined);
 
         const bindings: HttpBindingDefinition[] = [{
-                    authLevel: "anonymous",
-                    type: "httpTrigger",
-                    direction: "in",
-                    name: "req",
-                    route: args.route,
-                    methods: args.methods,
-                }, {
-                    type: "http",
-                    direction: "out",
-                    name: "$return",
-                }];
-
-        super("azure:appservice:HttpEventSubscription", name, bindings, {
+            authLevel: "anonymous",
+            type: "httpTrigger",
+            direction: "in",
+            name: "req",
+            route: args.route,
+            methods: args.methods,
+        }, {
+            type: "http",
+            direction: "out",
+            name: "$return",
+        }];

```

Comment:
in this case, can we indent one to help indicate these are children.  otherwise, it looks a bit wonky to me.

Reply:
restored original indentation in d783c138f08bff57325be4e7fb98e3a6933d5b8b